templates/primary-secondary/primary-secondary.js

```diff
@@ -49,6 +49,7 @@ class Resizer {
 		this.isMobile = false;
 		this.panelSize = 0;
 		this.isRtl = false;
+		this.secondaryFirst = false;
```

Comment:
This attribute isn't used for the Mobile Resizers so this could be added directly to the two Desktop Resizers but I found it a bit cleaner up here with `isRtl`